### ISSUE TITLE
Use robots.txt to reduce search engine traffic to older documentation.

### ISSUE THREAD

**Issue by wmwolf** [MESAHub/mesa#692]
We should use [robots.txt](http://www.robotstxt.org/robotstxt.html) to get search engines to guide users preferentially to the latest version of our documentation. Within the website, old versions will be unchanged, but web crawlers will be asked not to index those pages. I think it could be as simple as

```
User-agent: *
Disallow: /
Allow: /en/latest
```

We'd just need to get a file called `robots.txt` at the root directory of [docs.mesastar.org](http://docs.mesastar.org) with these contents (I think), and with time, the web crawlers should update their indexes.

**Comment by VincentVanlaer** [comment]
I faintly remembered that there was some way to refer indexers to the latest version of a page. I did some searching, and it seems that [the `canonical` link annotation](https://developers.google.com/search/docs/crawling-indexing/consolidate-duplicate-urls) can do this. From what I can tell this would be more effective, as robots.txt in the first place prevents crawling but not indexing. Meaning if someone links to an older version somewhere, these pages could still pop up. Although that's what I found from reading some stackexchange posts, so I don't know how correct this is.
